pitch.md

```diff
@@ -1,20 +1,32 @@
-# Project Title (replace me)
+# Bridge Hand DSL
 
 ## Describe your project in five words
+Easy, powerful, bridge hand language
 
 ## Audience
 _Who are you helping, with your language? What is that person's
 experience like now? How would their experience be better if you could help 
 them?_
 
+I'm helping bridge players.  This would help bridge players who want to study
+hands, share hands with other players, and just keep a database of hands. Also
+could help bridge organizations that help teach bridge players by providing an
+easy way to share hands with others.
```

Comment:
From class, I understand the audience and need of your language more, but I think it's also important to explain their experience now (either trying to explain hands through jargon or with the other language you mentioned) and why they aren't ideal.